README.md

```diff
@@ -1,3 +1,5 @@
 # Formalising Mathematics in Lean
 
-This is the course repo!
\ No newline at end of file
+This is the course repo. The course content (workshop sheets and homeworks) is located in the folder `ForMaL`,  which also contains some references we are using in the course.
+
+You can clone this repository to your own computer to work on things, if you already have lean installed. Or you can use codespaces by choosing the codespaces tab under the top right code menu, to try using lean in your web browser.
```

Comment:
```suggestion
## How to work on this

1. The easy method: Use Lean in your very own web browser!
   This is done by using Codespaces: you can find this option in the tab under the top right code menu.

2. Installing Lean on your computer following these [instructions](https://leanprover-community.github.io/get_started.html).
   Then you can clone this repository to your own computer by writing `leanprover get glams-lean-2024/formal-2024` in your terminal.
```

Reply:
should probably add a Codespaces link, and also a Gitpod link (instead of you navigating to the top right code menu and so on)